client/package.json

```diff
@@ -5,6 +5,7 @@
   "proxy": "http://localhost:5000/",
   "dependencies": {
     "@ant-design/icons": "^4.7.0",
+    "@ant-design/pro-table": "^2.74.4",
```

Comment:
I think you are not using this anymore?
If there's no one else using it you can uninstall it.
```suggestion
```